js/card.js

```diff
@@ -117,11 +117,11 @@ const createPopup = (offer, author) => {
 
   (offer.checkin && offer.checkout) ? addText(POPUP_TIME, `Заезд после ${offer.checkin}, выезд до ${offer.checkout}`) : hideElement(POPUP_TIME);
 
-  offer.features.length > 0 ? makePopupFeatures(POPUP_FEATURES, offer.features) : hideElement(POPUP_FEATURES);
+  (offer.features &&  offer.features.length > 0) ? makePopupFeatures(POPUP_FEATURES, offer.features) : hideElement(POPUP_FEATURES);
```

Comment:
```suggestion
  (Array.isArray(offer.features) &&  offer.features.length > 0) ? makePopupFeatures(POPUP_FEATURES, offer.features) : hideElement(POPUP_FEATURES);
```
то что значение featrues и в правду массив лучше проверять так.